ci/crd-ref-docs.nix

```diff
@@ -0,0 +1,34 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "crd-ref-docs";
+  version = "0.1.0";
+
+  # Don't run tests.
+  doCheck = false;
+  doInstallCheck = false;
+
+  src = fetchFromGitHub {
+    owner = "elastic";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Vp4HrT6Fj+stj2xasyz7i+UuogsFHeHOT8Ro0HeoGpI=";
+  };
+
+  subPackages = [
+    "."
+    "./config"
+  ];
+
+  vendorHash = "sha256-kNynNSa7xi5BL2oWBq7A2SWtjm1E+CyJd5f0FXkHAII=";
+
+  meta = with lib; {
+    description = "Generates Kubernetes CRD API reference documentation";
+    homepage = "https://github.com/joejulian/docker-tag-list";
+    license = licenses.asl20;
+    mainProgram = "docker-tag-list";
```

Comment:
`docker-tag-list` is generating CRD API documentation? I'm almost certain that `docker-tag-list` is returning you newest docker tag for particular container registry when regexp is provided.

Reply:
Fixed.